config_templates/gretel/transform/transform-nlp-off.yml

```diff
@@ -0,0 +1,33 @@
+# Policy to search for and redact "sensitive PII" as defined by `ask-experian` as well as
+# a custom defined regular expression for user IDs. 
+#
+# https://www.experian.com/blogs/ask-experian/what-is-personally-identifiable-information/
+
+# NLP is turned off using "use_nlp: false" for better performance. 
+# Set to true to improve detection or if dataset contains unstructured text.
+# https://docs.gretel.ai/classify/classify-model-configuration#classifying-data-using-nlp
+
+schema_version: "1.0"
+name: "redact-pii-nlp-off"
+models:
+  - transforms:
+      data_source: "_"
+      use_nlp: false
+      policies:
+        - name: remove_pii
+          rules:
+            - name: fake_or_redact_pii
+              conditions:
+                value_label:
+                  - person_name
+                  - credit_card_number
+                  - phone_number
+                  - us_social_security_number
+                  - email_address
+                  - location
+              transforms:
+                - type: fake
+                - type: redact_with_char
+                  attrs:
+                    char: X
```

Comment:
I think that for the configs that we use as our blueprints, we should change this `redact_with_char` to something else, like e.g. `hash` or `redact_with_label`.

`redact_with_char` can expose a bit too much information, like e.g. if only one person has a name with 2 letters in it, then after doing `redact_with_char` transform, you can still identify that person.

Reply:
done